xml/System/Double.xml

```diff
@@ -1946,7 +1946,7 @@ Euler's number is approximately 2.7182818284590452354.
         <Parameter Name="x" Type="System.Double" Index="0" FrameworkAlternate="net-7.0;net-8.0;net-9.0" />
       </Parameters>
       <Docs>
-        <param name="x">The power to which <code data-dev-comment-type="c">E</code> is raised.</param>
+        <param name="x">The power to which <c>E</c> is raised.</param>
```

Comment:
I don't think I've ever encountered _e_ in captial when referrring to Euler's number. The wiki URL has it that way, but the entirety of the text refers to it in italicized lower-case - https://en.wikipedia.org/wiki/E_(mathematical_constant). 

Looks like Wolfram does use `E` in Mathematica, so perhaps that is prior art to copy here. Just curious if this was discussed.

@tannergooding Was this an intentional choice?

Reply:
I'd have expected this to have been a `cref="E"`` so it refers to the named constant we use for `Euler's number`.

I know we have some support for `LaTeX` now, so if we're not doing a cref, then maybe using the "proper" math symbol is better?

CC. @gewarren

Reply:
Ah, I see. If this was referring to the .NET name, then that makes sense to me.